app/views/pages/supporters.html.erb

```diff
@@ -0,0 +1,42 @@
+<% @title = t('.title') %>
+
+<%= prose do %>
+  <p>
+    RubyGems.org is a critical part of the Ruby ecosystem, processing millions of downloads each day and enabling developers around the world to build with confidence. As a community-run service overseen by Ruby Central, a 501(c)(3) nonprofit, we rely on the support of individuals and organizations who believe in open source and want to keep RubyGems.org secure and reliable long-term.  Specifically, we fund this service through generous support from RubyGems Supporters, Ruby Central’s Corporate Sponsorship Program, Ruby Central memberships, and Infrastructure Sponsors.
+  </p>
+  <p>
+    The RubyGems Supporter Program is an easy way for you or your company to contribute to the maintenance, operations, and growth of the open source infrastructure you depend on. You can find out more about this program <a href="https://rubycentral.org/open-source/">here</a>.
+  </p>
+
+  <h2>Want to support RubyGems.org?</h2>
+  <p>
+    <a href="https://rubycentral.org/open-source/#/portal/signup">Become a Supporter</a>
+  </p>
+  <h2>RubyGems Supporters</h2>
+  <h3>Supporter Plus</h3>
+  <ul>
+    <li><a href="https://www.cloudamqp.com/">CloudAMQP's Logo</a></li>
+  </ul>
+
+  <h3>Supporter</h3>
+  <ul>
+    <li><a href="https://www.coffeeandcode.com/">Coffee and Code</a></li>
+  </ul>
+
+  <h2>Corporate Sponsors</h2>
+  <p>
+    Corporate sponsors support Ruby Central's work more broadly, including RubyGems.org and our other community initiatives.  There are opportunities starting at $10K/year for companies looking to support Ruby Central’s mission which includes RubyGems.org. You can learn more about Ruby Central’s Corporate Sponsorship Program <a href="https://rubycentral.org/corporate-sponsorship/">here</a>. 
+  </p>
+  <h2>Infrastructure Sponsors</h2>
+  <p>
+    Our infrastructure sponsors provide donated services that our team uses to keep RubyGems.org running smoothly and reliably, such as hosting, monitoring, security, and more.
+  </p>
+  <div class="w-full lg:grow mx-auto grid grid-cols-3 gap-6 justify-items-center md:flex md:items-center md:justify-between">
+    <a href="https://aws.amazon.com/" class="p-1"><%= icon_tag "aws", size: 15 %></a>
+    <a href="https://dnsimple.link/resolving-rubygems" class="p-1"><%= icon_tag "dnsimple", size: 15 %></a>
+    <a href="https://www.datadoghq.com/" class="p-1"><%= icon_tag "datadog", size: 15 %></a>
+    <a href="https://www.fastly.com/" class="p-1"><%= icon_tag "fastly", size: 15 %></a>
+    <a href="https://www.honeybadger.io/" class="p-1"><%= icon_tag "honeybadger", size: 15 %></a>
+    <a href="https://mend.io/" class="col-start-2 p-1"><%= icon_tag "mend-io", size: 15 %></a>
+  </div>
+<% end %>
```

Comment:
nitpicky, but can we add a new line at the end of the file? If you're using vscode, this can be configured to automatically happen - https://neilsonwebdesign.com/vs-code-force-newline-at-end-of-file.